### PR TITLE
Add Tagging documentation

### LINKED ISSUE #3696: TODO list for tagging
Ready for 3.6.0:
- [x] Align tags vertically in non-horizontal surfaces

For 3.7.0+:
- [x] Scripted API for custom tags, possibly like this #4146 
  ```
  ["Circle", "path\to\circle.paa"] call ace_tagging_fnc_createCustomTag
  ```
- [x] Custom tags by config documentation #4150 
- [x] Scripted API documentation #4150 
- [x] ~~Make checkTaggable public~~ not necessary

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Title - close #3696 
- Complements #4146 

**Merge when #4146 is released as stable!**

### CODE REVIEW

File: wiki/framework/tagging-framework.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+title: Tagging Framework
+description: Explains how to use tagging framework to add custom tags.
+group: framework
+order: 5
+parent: wiki
+---
+
+<div class="panel callout">
+    <h5>Note:</h5>
+    <p>This is just a guide to add tags using the framework, this is not a guide to make your own spray objects.</p>
```
Comment (bux578) [inline wiki/framework/tagging-framework.md]: `... framework. This is not ...`

File: wiki/framework/tagging-framework.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+</div>
+
+## 1. Config Values
+
+### 1.1 Adding a tag
+
+```cpp
+class ACE_Tags {
+    class yourTagClass {
+        displayName = "My Tag";  // Name of your tag as it displays in interaction menu
```
Comment (bux578) [inline wiki/framework/tagging-framework.md]: as it is displayed in the interaction menu

File: wiki/framework/tagging-framework.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        requiredItem = "ACE_SpraypaintBlack";  // Required item to have in inventory to be able to spray your tag (eg. `"ACE_SpraypaintBlack"`, `"ACE_SpraypaintRed"`, `"ACE_SpraypaintGreen"`, `"ACE_SpraypaintBlue"` or any custom item from `CfgWeapons`)
```
Comment (bux578) [inline wiki/framework/tagging-framework.md]: in the inventory

File: wiki/framework/tagging-framework.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        textures[] = {"path\to\texture1.paa", "path\to\texture2.paa"};  // List of texture variations (one is randomly selected when tagging)
+        icon = "path\to\icon.paa";  // Icon as it displays in interaction menu
```
Comment (bux578) [inline wiki/framework/tagging-framework.md]: Icon being displayed in the interaction menu

File: wiki/framework/tagging-framework.md (hunk 3 above)
Comment (jonpas) [inline wiki/framework/tagging-framework.md]: This doc is meant for other modders, I think config describes it better.

Comment (bux578) [inline wiki/framework/tagging-framework.md]: it's just the missing `the`

Comment (jonpas) [inline wiki/framework/tagging-framework.md]: Oh :facepunch:

**Comment by jonpas** [comment]
Replaced by #4260 for new wiki location.

**Pull request closed without merging**